x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/rule_params_modifier.ts

```diff
@@ -90,6 +92,23 @@ const applyBulkActionEditToRuleParams = (
         timelineTitle: action.value.timeline_title || undefined,
       };
       break;
+
+    // update look-back period in from and meta.from fields
+    case BulkActionEditType.set_schedule: {
+      const interval = parseInterval(action.value.interval) ?? moment.duration(0);
+      const parsedFrom = parseInterval(action.value.lookback) ?? moment.duration(0);
+
+      const from = parsedFrom.asSeconds() + interval.asSeconds();
+
+      ruleParams = {
+        ...ruleParams,
+        meta: {
+          ...ruleParams.meta,
+          from: action.value.lookback,
+        },
+        from: `now-${from}s`,
+      };
```

Comment:
It's obviously out of the scope of this PR, but I'm wondering why don't we simply have `interval` and `lookback` rule params and instead have these `from` and `meta.from` derivatives. @rylnd do you remember?

Reply:
Interested in knowing this, too.